libsrc/apple2/lseek.s

```diff
@@ -85,8 +85,12 @@ seek_common:
         bcs     oserr
 
         ; Need to return the position in EAX
+        .ifdef  __APPLE2ENH__
+        stz     sreg+1
+        .else
         lda     #0
         sta     sreg+1
+        .endif
```

Comment:
The Apple2 specific code uses `#$00` istead of `#0`. Please adjust the line above.